### PR TITLE
W-11791554: add resources paths in DeployableProjectModel

### PR DESCRIPTION
**Pull request by jcgrethe**

### CODE REVIEW

**Comment by jcgrethe** [comment]
--validate

**Comment by jcgrethe** [comment]
--validate

File: modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/deployable/DeployableProjectModel.java [hunk 1]
```diff
@@ -52,11 +53,14 @@ public final class DeployableProjectModel {
   private final Set<BundleDescriptor> sharedLibraries;
   private final Map<BundleDescriptor, List<BundleDependency>> additionalPluginDependencies;
 
+  private final List<Path> resourcesPath;
```
Comment (elrodro83) [inline modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/deployable/DeployableProjectModel.java:56]: isn't this just something intermediate for calculating the value for `resources`?

**Review by elrodro83 — CHANGES_REQUESTED** [review]

File: modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/deployable/DeployableProjectModel.java (hunk 1 above)
Comment (jcgrethe) [reply modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/deployable/DeployableProjectModel.java:56]: I will need it in the mule-framework to create the classloaders pointing to those resources folders

**Review by elrodro83 — APPROVED** [review]

**Pull request merged**